css/style.css

```diff
@@ -94,92 +89,73 @@ h3 {
   left: 22px;
   box-shadow: 0 6px 0 0 #000000, 0 12px 0 0 #000000;
 }
-.header .header_btns .search_btn {
+.header__search-btn {
   width: 60px;
   height: 60px;
   border: 1px solid #DDDDDD;
   background-color: #ffffff;
   position: relative;
 }
 @media (min-width: 1200px) {
-  .header .header_btns .navbar a {
+  .header__container {
+    height: 53px;
+  }
+  .header__navbar a {
     display: unset;
   }
-  .header .header_btns .menu_btn {
+  .header__burger-btn {
     display: none;
   }
-  .header .header_btns .search_btn {
+  .header__search-btn {
     width: 53px;
     height: 53px;
   }
 }
 
-.cards_list_container {
+.cards-list_container {
   margin-top: 121px;
   margin-bottom: 80px;
   margin-left: 17px;
   margin-right: 18px;
 }
 @media (min-width: 1200px) {
-  .cards_list_container {
+  .cards-list_container {
     width: 875px;
     margin-top: 80px;
     margin-left: 100px;
   }
 }
-.cards_list .card {
+
+.card {
   width: 340px;
   height: 355px;
   position: relative;
   border: 1px solid #DDDDDD;
```

Comment:
there's something wrong with the borders, they don't follow the designs

<img width="655" alt="Screenshot 2024-08-23 at 12 22 16 PM" src="https://github.com/user-attachments/assets/9fe5e523-1b11-4973-9f67-9f82ea12e600">

<img width="734" alt="Screenshot 2024-08-23 at 12 23 32 PM" src="https://github.com/user-attachments/assets/e64f71f9-3e94-49d0-89b9-e7133117f2ee">